cmd/storage_provider/init.go

```diff
@@ -91,6 +91,24 @@ func initResourceManager(ctx *cli.Context) error {
 	return nil
 }
 
+// initPprof initializes global pprof service for performance monitoring.
+func initPprof(ctx *cli.Context, cfg *config.StorageProviderConfig) error {
```

Comment:
Pls normalize `PProf` and `Pprof` into one style.

Reply:
fixed